src/utils/mod.rs

```diff
@@ -648,28 +648,71 @@ impl TryFrom<Signature> for TPMT_SIGNATURE {
 
 /// Rust native wrapper for session attributes objects.
 #[derive(Copy, Clone, Debug, Default)]
-pub struct TpmaSession(TPMA_SESSION);
+pub struct TpmaSession {
```

Comment:
Just a small question: this structure seems to represent session attributes, and is name `TpmaSession`. The variable name holding this structure value is often "attributes" or "session_attrs". Could we rename it `TpmaSessionAttributes`? Is it to match with the name it is given in the TSS?

Reply:
I guess we could, the name was initially based on the TSS structure, but since we're doing something more abstract now, we can drop the TSS naming convention (it might even be confusing if we use the same name but offer different functionality)

Reply:
I agree. But most of the names in the code is using the TSS names. But it would make sense to rename it to something like SessionAttributes instead.

Reply:
I don't mind you doing it in this PR or another one 😃 Or just raising an issue.